cryptol-remote-api/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Revision history for `cryptol-remote-api` and `cryptol-eval-server`
 
-## 2.13.0 -- YYYY-MM-DD
+## 2.13.2 -- YYYY-MM-DD
```

Comment:
Nit pick: we should use something like `next` here as the version number until we actually bump the version number in `cryptol-remote-api.cabal` itself.